tools/codegen/gen_backend_stubs.py

```diff
@@ -264,5 +280,49 @@ def make_file_manager(install_dir: str) -> FileManager:
                 )),
             })
 
+        # generate native function impls that build IR nodes
+        for dispatch_key in [backend_dispatch_key]:  # , autograd_dispatch_key
+            fm.write_with_template(f'{dispatch_key}NativeFunctions.cpp', 'NativeFunctions.cpp', lambda: {
+                'generated_comment' : '', 
+                'includes': [f'#include "{path}"' for path in [
+                    "lazy_tensor_core/csrc/tensor.h",
+                    "lazy_tensor_core/csrc/aten_ltc_bridge.h",
+                ]],
+                'native_functions_include': f'#include "{output_dir}/{backend_key}NativeFunctions.h"',
+                'backend_namespace': 'torch_lazy_tensors',  # this is wrong
+                'native_function_definitions': 
+                list(concatMap(
+                    lambda f: dest.compute_lazy_native_function_definition(f, backend_indices[CODEGEN_MAGIC_NUMBER]),
```

Comment:
Yea, i definitely won't ship the hacky magic number.  But I wanted to get suggestions on a better way.  

I think I could make parse_backend_yaml return a separate list, and maybe figure out how to pass that list to my generators instead of using the backend index directly. 

Does that suggestion apply equally to the case I currently have in yaml where there is a separate list of ops for the new codegen vs the existing 'backend' list for the old codegen?  Or is this suggestion mainly for the case where i made it a 'flag' for each op whether to opt-in to the more complete codegen?